.github/workflows/helm-chart-smoketest.yml

```
@@ -4,71 +4,133 @@ on:
   workflow_call:
 
 env:
-  SHIM_SPIN_VERSION: v0.15.1
+  K8S_VERSION: v1.31.2
+  MICROK8S_CHANNEL: 1.31/stable
+  SHIM_SPIN_VERSION: v0.17.0
```

Lets update those versions
```suggestion
  K8S_VERSION: v1.32.2
  MICROK8S_CHANNEL: 1.32/stable
  SHIM_SPIN_VERSION: v0.18.0
```